jujugui/static/gui/src/app/components/deployment/summary/change-item/_change-item.scss

```diff
@@ -0,0 +1,28 @@
+.deployment-summary-change-item {
```

Comment:
Note for reviewer: this component is a duplicate of the existing change-item scss.